path-animate/demo.html

```diff
@@ -97,19 +97,32 @@
         stage.update();
     }
 
+    function distanceBetweenPoints(xDist, yDist) {
+
+        dist = Math.sqrt(Math.pow(xDist, 2) + Math.pow(yDist, 2));
+
+        return dist;
```

Comment:
```suggestion
        return Math.sqrt(Math.pow(xDist, 2) + Math.pow(yDist, 2));
```

Reply:
the temp variable is never used, so it is more efficient and clearer code to just return the expression.